### DISCUSSION TITLE
使用UDP协议遇到问题

### DISCUSSION THREAD

**Discussion by zaiqixiao** [MikeWang000000/Natter#78] [Q&A]
![QQ截图20240411231555](https://github.com/MikeWang000000/Natter/assets/166514485/1206c8fd-80d5-4195-9b68-62fad6d17506)
![QQ截图20240411231635](https://github.com/MikeWang000000/Natter/assets/166514485/f5ccc2f6-f9ed-4f3b-a18f-e41e21bbdf8b)
![QQ截图20240411231720](https://github.com/MikeWang000000/Natter/assets/166514485/ee7a07d5-d508-443f-b331-edfa4c9bb33b)
![QQ截图20240411231737](https://github.com/MikeWang000000/Natter/assets/166514485/bb109992-374a-4149-8c11-599ebd027d3b)
TCP正常，但是UDP会卡在这里

**Comment by MikeWang000000** [comment]
正常哈，因为没有办法精确判定 UDP 端口的开放状态，UDP 模式下所以不会输出检测信息。
使用 `-v` 打开详细输出，你会发现是正常运行的。

**Reply by zaiqixiao** [reply]
谢谢佬已经解决了，我看udp没有回包以为是打洞没成功。
